template/src/ontology/Makefile.jinja2

```diff
@@ -219,15 +325,24 @@ imports/%_terms_combined.txt: seed.txt imports/%_terms.txt
 # This pattern uses ROBOT to generate an import module
 imports/%_import.owl: mirror/%.owl imports/%_terms_combined.txt
 	$(ROBOT) extract -i $< -T imports/$*_terms_combined.txt --method BOT \
-		 annotate --ontology-iri $(ONTBASE)/$@ --version-iri $(ONTBASE)/releases/$(TODAY)/$@ --output $@
+		annotate --ontology-iri $(ONTBASE)/$@ --version-iri $(ONTBASE)/releases/$(TODAY)/$@ --output $@
 .PRECIOUS: imports/%_import.owl
 
+{% if 'obo' in project.export_formats -%}
 # convert imports to obo.
 # this can be useful for spot-checks and diffs.
 # we set strict mode to false by default. For discussion see https://github.com/owlcs/owlapi/issues/752
 imports/%_import.obo: imports/%_import.owl
-	$(ROBOT) convert --check false -i $< -f obo -o $@.tmp && mv $@.tmp $@
```

Comment:
Is this deliberate? This is dangerous as convert can fail leaving an empty file. I always use the $@.tmp paradigm, even if it is ugly. (For consistency now I do `$@.tmp.obo` since ROBOT uses suffix as default)

Reply:
THATS why you had this in!!! Ahhh! Thanks for pointing this out, I put it back now! :)